arch/arm/src/sama5/sam_gmac.c

```diff
@@ -1642,14 +1657,13 @@ static int sam_gmac_interrupt(int irq, void *context, FAR void *arg)
    *   one to this bit.
    */
 
-  tsr = sam_getreg(priv, SAM_GMAC_TSR_OFFSET);
+  tsr = sam_getreg(priv, SAM_GMAC_TSR);
```

Comment:
This is the fix for the txtimeout always firing; the old code had the wrong register address here.